assets/js/modules/analytics/datastore/properties.test.js

```diff
@@ -123,6 +123,67 @@ describe( 'modules/analytics properties', () => {
 				expect( properties ).toEqual( undefined );
 			} );
 		} );
+		describe( 'selectProperty', () => {
+			it( 'requires a valid propertyID', () => {
+				expect( () => {
+					registry.dispatch( STORE_NAME ).selectProperty();
+				} ).toThrow( 'A valid propertyID selection is required.' );
+			} );
+
+			it( 'returns if the accountID is not set', () => {
+				fetchMock.get(
+					/^\/google-site-kit\/v1\/modules\/analytics\/data\/properties-profiles/,
+					{ body: fixtures.propertiesProfiles, status: 200 }
+				);
+				const propertyID = fixtures.propertiesProfiles.properties[ 0 ].id;
+				expect( registry.select( STORE_NAME ).getPropertyID() ).toBeUndefined();
+				registry.dispatch( STORE_NAME ).selectProperty( propertyID );
+				expect( registry.select( STORE_NAME ).getPropertyID() ).toBeUndefined();
+			} );
+
+			it( 'selects the property', async () => {
```

Comment:
Let's be a little more clear as to what is happening in this test as it asserts more than just the property selection 👍 
```suggestion
			it( 'selects the property and its default profile when set', async () => {
```